browser/playlist/test/playlist_download_request_manager_browsertests.cc

```diff
@@ -83,21 +85,30 @@ class PlaylistDownloadRequestManagerBrowserTest : public PlatformBrowserTest {
         std::make_unique<playlist::MediaDetectorComponentManager>(nullptr);
     component_manager_->SetUseLocalScriptForTesting();
 
+    auto* profile = chrome_test_utils::GetProfile(this);
     request_manager_ =
         std::make_unique<playlist::PlaylistDownloadRequestManager>(
-            chrome_test_utils::GetProfile(this), component_manager_.get());
+            profile, HostContentSettingsMapFactory::GetForProfile(profile),
+            component_manager_.get());
   }
 
   void TearDownOnMainThread() override {
     request_manager_.reset();
     component_manager_.reset();
 
-    ASSERT_TRUE(embedded_test_server()->ShutdownAndWaitUntilComplete());
+    if (embedded_test_server()->Started())
+      ASSERT_TRUE(embedded_test_server()->ShutdownAndWaitUntilComplete());
```

Comment:
never seen that before. is there a reason for this?

Reply:
@sangwoo108 I deleted `ASSERT_TRUE(embedded_test_server()->ShutdownAndWaitUntilComplete());`.
As new test case(`PlaylistDownloadRequestManagerBrowserTest.BackgroundWebContents`)doesn't start server, it caused crash at teardown. I think all other tests works well w/o this check. WDYT?